openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -601,6 +614,17 @@ def login_user(request, api_version='v1'):
                 enterprise_selection_page(request, possibly_authenticated_user, finish_auth_url or next_url)
             )
 
+        if (
+            settings.ENABLE_AUTHN_LOGIN_HIBP_NUDGE and
+            0 < password_frequency <= settings.LOGIN_HIBP_NUDGE_FREQUENCY_THRESHOLD
+        ):
```

Comment:
we don't need to add another condition here. We are already giving a default value to pwned_properties here:
https://github.com/openedx/edx-platform/blob/a3eca17db678723487203c043171d561aa841923/openedx/core/djangoapps/user_authn/views/login.py#L585-L588

and password_frequency will be 0 (zero). 
```
0 < password_frequency <= settings.LOGIN_HIBP_NUDGE_FREQUENCY_THRESHOLD
```
This will work for nudge as already added ^ and for block 0 < blocking threshold.